### ISSUE TITLE
Media not importing on branch ui-improvements

### ISSUE THREAD

**Issue by mebrett** [omeka-s-modules/CSVImport#130]
Tried importing items and resources, things were created but the media didn't seem to come through at all (no media shows up at all for items)

**Comment by kimisgold** [comment]
Here's the test case I ran:

[**Birbs3.csv**](https://www.dropbox.com/s/lr1tvb1mjjkgiwu/BirbsThree.csv?dl=0)

<img width="1392" alt="screen shot 2018-03-20 at 4 52 24 pm" src="https://user-images.githubusercontent.com/606764/37682214-41825ef4-2c5f-11e8-82de-9c4f11cfdefc.png">

From this, I was able to successfully import 1 item set, 1 item, and 1 media, with no error messages and the media attached to the appropriate item.

@zerocrates noticed that if you ran the same import multiple times without undoing past import, subsequent media would attach to the first version of the item—the media is using the item title as its identifier, and thus gloms onto the first item with that title it finds. See the duplicate media in the first version of "Cooper's Hawk" below.

<img width="1026" alt="screen shot 2018-03-20 at 4 56 53 pm" src="https://user-images.githubusercontent.com/606764/37682355-c2434e2c-2c5f-11e8-9824-ba89e1e299a3.png">

<img width="1392" alt="screen shot 2018-03-20 at 4 57 08 pm" src="https://user-images.githubusercontent.com/606764/37682370-c830eace-2c5f-11e8-9ce3-dde662b2a3e6.png">
